### DISCUSSION TITLE
Is there any chance to make a release soon?

### DISCUSSION THREAD

**Discussion by terlar** [dorssel/usbipd-win#638] [Q&A] [answered]
I am referencing the auto-attach.sh script when packaging it up in a systemd service and would like to have a more stable reference than a "random" hash. Is there a plan for a release soon, could be just a minor one. Hopefully it would not be too much trouble, thanks!

**Comment by dorssel** [comment]
I'm waiting for confirmation that the dreaded BSOD issue is resolved. If it is, then a release will follow shortly.

**Reply by terlar** [reply]
Ah, can I help to verify this? I think I have run into that at least 3 times. Wasn't sure what caused it, but pretty sure it was related to WSL and also potentially USB. So I might have been hit by that issue

**Reply by dorssel** [reply]
Try to reproduce the BSOD with 3.0.0, at least somewhat consistently. Then install the latest master build from https://github.com/dorssel/usbipd-win/actions/runs/5261546787 and see if it no longer happens. (Un)fortunately, I myself have never experienced the BSOD. It seems to happen when flashing large adb images? See issues.

**Comment by dorssel** [comment, marked as answer]
We are now at 3.1.0 and BSOD has been resolved. Also, the updated auto-attach script is in it.
